plugins/widget/plugin.js

```diff
@@ -1534,9 +1540,9 @@
 			editor.fire( 'lockSnapshot' );
 			this.dragHandlerContainer.setStyles( {
 				top: newPos.y + 'px',
-				left: newPos.x + 'px',
-				display: 'block'
+				left: newPos.x + 'px'
 			} );
+			this.dragHandlerContainer.removeClass( '.cke_widget_drag_handler_container_hidden' );
```

Comment:
After removing this class this line won't be needed, but please note that `removeClass` takes class name **without** dot at the beginning.